docker-compose.override.yaml

```diff
@@ -11,21 +11,17 @@ services:
         scripts/wait-for/wait-for ffc-demo-claim-message-queue:8161 --
         npm run migrate &&
         npm run start:watch"
-    ports:
-      - "3003:3003"
     depends_on:
       - ffc-demo-claim-message-queue
-    networks:
-      - ffc-demo-claim
+    ports:
+      - "3003:3003"
 
   ffc-demo-claim-postgres:
-    networks:
-      - ffc-demo-claim
     volumes:
       - postgres_data:/var/lib/postgresql/data
 
   ffc-demo-claim-message-queue:
-    container_name: ffc-demo-claim-message-queue
+    container_name: ffc-demo-claim-message-queue-${BUILD_NUMBER:-local}

```

Comment:
Adding the build number or `-local` suffix has no real consequence as this message queue isn't run by the pipelines, but it does make the container name consistent with sibling containers in development.